en/dronecan/raccoonlab_nodes.md

```diff
@@ -0,0 +1,67 @@
+# RaccoonLab Mini/Micro Nodes
+
+Mini/Micro are general purpose CAN nodes. The functionality of these devices depends on the firmware they are running.
+The default and most popular usage is as a PWM-CAN adapter to control ESCs and servos using CAN (Cyphal or DroneCAN) communication:
+
+- Micro is the smallest node with 2 groups of pins (PWM/5V/GND) to control 2 servos or ESCs.
+- Mini v2: Features 2 x PWM/5V/GND for controlling servos or ESCs and 2 x PWM/FB/GND for controlling and receiving feedback from ESCs via UART.
+
+## Cyphal/DroneCAN CAN-PWM Adapter Firmware
+
+_Cyphal/DroneCAN CAN-PWM_ firmwares are the default for Mini and Micro nodes.
```

Comment:
Very cool. I guess one concern is that CAN ESC are very precise and don't need calibration in the same way as PWM ones do. I presume you still need to do this "somewhere" so that should be noted in the docs - your docs :-)